src/pudl/package_data/ferc1/xbrl_factoid_rate_base_category_tags.csv

```diff
@@ -0,0 +1,217 @@
+core_ferc1__yearly_plant_in_service_sched204,office_furniture_and_equipment_general_plant,,,,general_plant
+core_ferc1__yearly_plant_in_service_sched204,transportation_equipment_general_plant,,,,general_plant
+core_ferc1__yearly_plant_in_service_sched204,stores_equipment_general_plant,,,,general_plant
+core_ferc1__yearly_plant_in_service_sched204,tools_shop_and_garage_equipment_general_plant,,,,general_plant
+core_ferc1__yearly_plant_in_service_sched204,laboratory_equipment_general_plant,,,,general_plant
+core_ferc1__yearly_plant_in_service_sched204,power_operated_equipment_general_plant,,,,general_plant
+core_ferc1__yearly_plant_in_service_sched204,communication_equipment_general_plant,,,,general_plant
+core_ferc1__yearly_plant_in_service_sched204,miscellaneous_equipment_general_plant,,,,general_plant
+core_ferc1__yearly_plant_in_service_sched204,other_tangible_property_general_plant,,,,general_plant
+core_ferc1__yearly_plant_in_service_sched204,asset_retirement_costs_for_general_plant_general_plant,,,,general_plant
+core_ferc1__yearly_utility_plant_summary_sched200,abandonment_of_leases,,,,electric_plant_leased_to_others
+core_ferc1__yearly_utility_plant_summary_sched200,amortization_and_depletion_utility_plant_leased_to_others,,,,electric_plant_leased_to_others
+core_ferc1__yearly_utility_plant_summary_sched200,amortization_of_other_utility_plant_utility_plant_in_service,,,,other_electric_plant
+core_ferc1__yearly_utility_plant_summary_sched200,amortization_of_plant_acquisition_adjustment,,,,other_electric_plant
+core_ferc1__yearly_utility_plant_summary_sched200,amortization_utility_plant_held_for_future_use,,,,electric_plant_held_for_future_use
+core_ferc1__yearly_utility_plant_summary_sched200,construction_work_in_progress,,,,construction_work_in_progress
+core_ferc1__yearly_utility_plant_summary_sched200,depreciation_amortization_and_depletion_utility_plant_leased_to_others,,,,electric_plant_leased_to_others
+core_ferc1__yearly_utility_plant_summary_sched200,depreciation_and_amortization_utility_plant_held_for_future_use,,,,electric_plant_held_for_future_use
+core_ferc1__yearly_utility_plant_summary_sched200,depreciation_utility_plant_held_for_future_use,,,,electric_plant_held_for_future_use
+core_ferc1__yearly_utility_plant_summary_sched200,depreciation_utility_plant_leased_to_others,,,,electric_plant_leased_to_others
+core_ferc1__yearly_utility_plant_summary_sched200,depreciation_utility_plant_in_service,other,,,net_utility_plant
+core_ferc1__yearly_utility_plant_summary_sched200,depreciation_utility_plant_in_service,gas,,,net_utility_plant
+core_ferc1__yearly_utility_plant_summary_sched200,depreciation_utility_plant_in_service,common,,,net_utility_plant
+core_ferc1__yearly_utility_plant_summary_sched200,depreciation_utility_plant_in_service,other3,,,net_utility_plant
+core_ferc1__yearly_utility_plant_summary_sched200,depreciation_utility_plant_in_service,other2,,,net_utility_plant
```

Comment:
this is the only tag from @jrea-rmi that i needed to edit a little bit. the `utility_type=="electric"` version of this factoid is linked to other more granular facts that have their own `rate_base_category` that are more specific than `net_utility_plant`. This was causing the error `Calculation forest nodes have conflicting tags:` so i removed the generic category and labeled only the non-electric utility types.